src/DevBetterWeb.Infrastructure/Data/AppDbContext.cs

```diff
@@ -22,12 +22,16 @@ public class AppDbContext : DbContext
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             base.OnModelCreating(modelBuilder);
+
+            modelBuilder.Entity<Book>().Property(b => b.Title).HasMaxLength(100);
```

Comment:
This belongs in a config class in the config folder.

Reply:
Actually you already have `/Config/BookConfig.cs` so this should be removed.

Reply:
removed